<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Commander</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="leftPanel">
        <property name="title">
         <string>Local Drive</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="driveComboLeft"/>
         </item>
         <item>
          <widget class="QTreeView" name="leftView">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="copyButton">
          <property name="text">
           <string>Copy →</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveButton">
          <property name="text">
           <string>Move →</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newFolderButton">
          <property name="text">
           <string>New Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="renameButton">
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uploadButton">
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="downloadButton">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="rightPanel">
        <property name="title">
         <string>Remote/Other Drive</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="driveComboRight"/>
         </item>
         <item>
          <widget class="QTreeView" name="rightView">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="networkGroup">
      <property name="title">
       <string>Network Transfer</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Server Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serverAddressEdit">
         <property name="placeholderText">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serverPortEdit">
         <property name="placeholderText">
          <string>12345</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startServerButton">
         <property name="text">
          <string>Start Server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transferFileButton">
         <property name="text">
          <string>Transfer File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>